liquibase-core/src/test/java/liquibase/database/DatabaseFactoryTest.java

```diff
@@ -93,7 +96,11 @@ public void openConnectionLoadsDriverPropertiesFromGivenFile() throws Exception
         }
         String propsFilePath = propsFile.getAbsolutePath();
 
-        DatabaseConnection dbConnection = databaseFactory.openConnection("jdbc:h2:mem:DatabaseFactoryTest", "sa", "", null, null, propsFilePath, null, resourceAccessor);
+        Map<String, String> resources = new HashMap<>();
+        resources.put(propsFilePath, propsFile.toString());
+        ResourceAccessor mockAccessor = new MockResourceAccessor(resources);
```

Comment:
Unsure if this is the best approach but `mock(ResouceAccessor.class)` doesn't have anything populated so this test fails since I changed the buildDriverProperties method to actually use a ResourceAccessor.

Reply:
Irrelevant with different approach